### ISSUE TITLE
Error with choices=BOOLEANS

### ISSUE THREAD

**Issue by martialt** [synthesio/infra-ovh-ansible-module#6]
Hi,
I used this module and I raised an error when I used it with.
```
- name: Add server IP to DNS
  ovh: service='dns' domain='mydomain.xyz' ip='111.222.333.444' name='appli-name'
```
`<111.222.333.444> (0, 'Traceback (most recent call last):\r\n  File "/tmp/ansible_iy0dE9/ansible_module_ovh.py", line 543, in <module>\r\n    main()\r\n  File "/tmp/ansible_iy0dE9/ansible_module_ovh.py", line 503, in main\r\n    force_reboot = dict(required=False, default=\'no\', choices=BOOLEANS),\r\nNameError: global name \'BOOLEANS\' is not defined\r\n', 'Shared connection to 111.222.333.444 closed.\r\n')
fatal: [111.222.333.444]: FAILED! => {
    "changed": false, 
    "module_stderr": "Shared connection to 111.222.333.444 closed.\r\n", 
    "module_stdout": "Traceback (most recent call last):\r\n  File \"/tmp/ansible_iy0dE9/ansible_module_ovh.py\", line 543, in <module>\r\n    main()\r\n  File \"/tmp/ansible_iy0dE9/ansible_module_ovh.py\", line 503, in main\r\n    force_reboot = dict(required=False, default='no', choices=BOOLEANS),\r\nNameError: global name 'BOOLEANS' is not defined\r\n", 
    "msg": "MODULE FAILURE", 
    "rc": 0
}
`

I changed 
`force_reboot = dict(required=False, default='no', choices=BOOLEANS),`
by
`force_reboot = dict(required=False, default='no', type='bool'),`

And now it works.

Regards,

**Comment by Skunnyk** [comment]
Should be fixed by https://github.com/synthesio/infra-ovh-ansible-module/pull/10

**Comment by Skunnyk** [comment]
Fixed in v2.0.
